zypp/target/TargetImpl.cc

```diff
@@ -1541,12 +1543,39 @@ namespace zypp
 
       bool abort = false;
 
+      bool singleTrans = false;
```

Comment:
Here its possible to switch between new and old path , we could use a env var for that